protocol/x/listing/keeper/grpc_query_markets_hard_cap_test.go

```diff
@@ -0,0 +1,42 @@
+package keeper_test
+
+import (
+	"testing"
+
+	"github.com/dydxprotocol/v4-chain/protocol/x/listing/types"
+	"github.com/stretchr/testify/require"
+
+	testapp "github.com/dydxprotocol/v4-chain/protocol/testutil/app"
+)
+
+func TestQueryPMLEnabledStatus(t *testing.T) {
+	tests := map[string]struct {
+		hardCap uint32
+	}{
+		"Hard cap - 0": {
+			hardCap: 0,
+		},
+		"Hard cap - 100": {
+			hardCap: 100,
+		},
+	}
+
+	for name, tc := range tests {
+		t.Run(
+			name, func(t *testing.T) {
+				tApp := testapp.NewTestAppBuilder(t).Build()
+				ctx := tApp.InitChain()
+				k := tApp.App.ListingKeeper
+
+				// set permissionless listing to true for test
```

Comment:
nit: update comments to refer to this as changing the cap rather than setting to true / false.